### PR TITLE
fix HTML nits

### PR DESCRIPTION
**Pull request by retokromer**
- unify `<br/>` to `<br>`
- update date

### CODE REVIEW

File: manual/genManual.cpp [hunk 1]
```diff
@@ -66,7 +66,7 @@ static void addHeader(QString& out)
              "<html>\n"
              "<head>\n"
              "   <meta charset=\"utf-8\">\n"
-             "   </head>\n"
+             "</head>\n"
```
Comment (Jojo-Schmitz) [inline manual/genManual.cpp]: This is our house style, "banner style"

File: manual/genManual.cpp [hunk 2]
```diff
@@ -76,7 +76,7 @@ static void addHeader(QString& out)
 
 static void addFooter(QString& out)
       {
-      out += /* "<div class=\"footer\"><a href=\"https://musescore.org/\">MuseScore</a> - Free music notation software<br/>\n"
+      out += /* "<div class=\"footer\"><a href=\"https://musescore.org/\">MuseScore</a> - Free music notation software<br>\n"
              "&copy; 2002-2016 Werner Schweer &amp; others</div>\n" */
```
Comment (Jojo-Schmitz) [inline manual/genManual.cpp]: 2019

File: manual/genManual.cpp (hunk 1 above)
Comment (retokromer) [reply manual/genManual.cpp]: Weird, yet okay then.

File: manual/genManual.cpp (hunk 2 above)
Comment (retokromer) [reply manual/genManual.cpp]: done

**Comment by RobFog** [comment]
Please squash the two commits.

**Comment by retokromer** [comment]
Hmm… Visibly not what I expected!

**Comment by Jojo-Schmitz** [comment]
I use `git pull --rebase upstream master` for this. Never use `git merge` here

**Comment by retokromer** [comment]
> I use `git pull --rebase upstream master` for this.

It’s what I do usually, but I did understand from the indicated link that something different was needed here…

**Pull request closed without merging**